src/DacpacTool/DacpacTool.csproj

```diff
@@ -13,8 +13,8 @@
 
   <ItemGroup>
     <!-- These packages contain DacFX analysis rules, which must be located with the executable in order to be discovered-->
-    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="1.0.0" IncludeAssets="runtime" />
-    <PackageReference Include="ErikEJ.DacFX.TSQLSmellSCA" Version="1.0.0" IncludeAssets="runtime" />
+    <PackageReference Include="ErikEJ.DacFX.SqlServer.Rules" Version="[1.0,2.0)" IncludeAssets="runtime" />
```

Comment:
Just annotating this for reference: per [this documentation](https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort), this format means `1.0 ≤ x < 2.0`

Reply:
Thanks for the thoughtful review!  :-)